### DISCUSSION TITLE
Priorities -

### DISCUSSION THREAD

**Discussion by smartopian** [KantaraInitiative/ancr-wg#10] [General]
@IDmachines  I think for Kay to join the advisory effort, we would need to propose the ANCR WG new Chartered proposal, to work on conformance  in Kantara - and move standard to tDigital Governance Council, then ask if Kay would be interested in Advising on the development of this program, with a view to representing Kantara in a future - 1.5yr from now Conformity assessment scheme for this.  

What's the front door looking like ?  Does this go to the LC and the Kantara Board?    And Should this be what we are working on this week ?

**Comment by IDmachines** [comment]
It would be something to present to Kay and try to make it a simple ask. I would then be up to Kay how to proceed (yes/no), understand the budget, etc. requirements.

 

There is no need to do this until there is a draft of a conformance program in my opinion.

 

There is nothing to do on Kantara’s part to move the work, As I have mentioned the Kantara IP kicks in when something is published and governs work in the workgroup gear toward those publications.

 

From: Mark Lizar ***@***.***> 
Sent: Monday, August 14, 2023 2:46 PM
To: KantaraInitiative/WG-ANCR ***@***.***>
Cc: Salvatore D'Agostino ***@***.***>; Mention ***@***.***>
Subject: [KantaraInitiative/WG-ANCR] Priorities - (Discussion #10)

 

@IDmachines <https://github.com/IDmachines>  I think for Kay to join the advisory effort, we would need to propose the ANCR WG new Chartered proposal, to work on conformance in Kantara - and move standard to tDigital Governance Council, then ask if Kay would be interested in Advising on the development of this program, with a view to representing Kantara in a future - 1.5yr from now Conformity assessment scheme for this.

Does this go to the LC and the Kantara Board? And Should this be what we are working on this week ?

**Reply by smartopian** [reply]
well, i guess it comes down to when Kantara is included - before it is proposed to DGC or after.  If Kay is invited in after than we are doing this without Kantara invovled up front
